TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/dialog/TweetDialog.java

```diff
@@ -246,7 +247,7 @@ private void setMenuItems(MenuArrayAdapter adapter) {
         if (menuSetting.contains(DELETE)) {
             if (mSource.isMyTweet() && !mStatus.isRetweet()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1066)

Reply:
Priority is given to code readability.